auto_route/example/analysis_options.yaml

```diff
@@ -1,4 +1,4 @@
-include: package:pedantic/analysis_options.yaml
+# include: package:pedantic/analysis_options.yaml
```

Comment:
Generally speaking, it's better to have a linter in place to improve the code while developing.
Was there any particular reason that brought you to comment this line out?

Reply:
FYI, IIRC this line serves no actual purpose in the codebase save for making the parent linter complain.